assets/css/about-calibration.css

```
@@ -768,80 +1087,135 @@ html[lang="zh"] .studio-interview__intro h2 {
+
+  .studio-interview:has(.studio-interview__native:not([data-state="idle"])) .studio-interview__intro {
+    display: none;
   }
 }
 
 @media (max-width: 640px) {
   .studio-interview {
-    --tp-embed-min-height: 390px;
-    gap: 24px;
+    --studio-interview-min: 390px;
+    gap: 20px;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)